easybuild/easyconfigs/b/Bowtie/Bowtie-1.1.2-intel-2015a.eb

```diff
@@ -0,0 +1,30 @@
+#Modified from existing version by:
+#Robert Schmidt <rjeschmi@gmail.com>
+#Ottawa Hospital Research Institute - Bioinformatics Team
+easyblock = 'MakeCp'
+
+name = 'Bowtie'
+version = '1.1.2'
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+It aligns short DNA sequences (reads) to the human genome.
+"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = ['%(namelower)s-%(version)s-src.zip']
+source_urls = ['http://download.sourceforge.net/bowtie-bio/']
+
+patches = ['compiler-agnostic.patch', 'int64typedef.patch']
```

Comment:
I would prefer using `buildopts` rather than including a patch to remove the compiler variables from the Makefile (a patch will break sooner or later):

```python
buildopts = 'CC="$CC" CXX="$CXX" CPP="$CXX"'
```